### DISCUSSION TITLE
using the output

### DISCUSSION THREAD

**Discussion by Sublevel4** [mebjas/html5-qrcode#217] [Q&A]
I am not that great at JS so I am learning as I go so I apologize for the basic question. If I want to use the output from the scan that appears below the scanner, how would I go about that? what am I looking for?

**Comment by ROBERT-MCDOWELL** [comment]
var scanner = new Html5QrcodeScanner("scanner-element",{fps:10,qrbox:250});
scanner.render(onScanSuccess,onScanFailure);
function exchange_onScanSuccess(data){
    // data is the result from scanner
}
